docs/logs.md

```diff
@@ -48,3 +48,33 @@ LOGGING = {
     },
 }
 ```
+
+### IgnoreIfContains
+
+Ignore log record if message entry contains any pattern set on log filter configuration.
+
+In order to use it, include a filter entry in your logging dictconfig.
+
+#### Example
+
+```python
+LOGGING = {
+        'version': 1,
+        'filters': {
+            'ignore_if_contains': {
+                '()': 'django_toolkit.logs.filters.IgnoreIfContains',
+                'patterns': ['/ping', '/healthcheck'],
```

Comment:
What do you think about using `substring` instead of `pattern`? Because term pattern is usually used for regular expressions.